homeassistant/components/tplink/camera.py

```diff
@@ -108,7 +108,7 @@ def __init__(
 
     def _get_unique_id(self) -> str:
         """Return unique ID for the entity."""
-        return f"{legacy_device_id(self._device)}-{self.entity_description}"
```

Comment:
This is currently only on dev and was not released yet, correct?

Reply:
Would you mind adding it to the beta milestone?

Reply:
The beta has not been cut yet, so if it is merged now it should be included in the beta

Reply:
Would you be able to merge it for me please?